src/main/java/org/sagebionetworks/web/client/jsinterop/SxProps.java

```diff
@@ -0,0 +1,29 @@
+package org.sagebionetworks.web.client.jsinterop;
+
+import jsinterop.annotations.JsOverlay;
+import jsinterop.annotations.JsPackage;
+import jsinterop.annotations.JsType;
+
+@JsType(isNative = true, namespace = JsPackage.GLOBAL, name = "Object")
+public class SxProps {
+
+  private String color;
+  private long fontWeight;
```

Comment:
Can add more properties as needed

Reply:
Maybe add this comment to the code instead (with a couple of SxProps examples)?